### PR TITLE
Configures robots.txt

### PR DESCRIPTION
**Pull request by jasonKufner**
BP-1291

### CODE REVIEW

File: apps/mainsite/static/robots.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# www.robotstxt.org/
-# https://developers.google.com/webmasters/control-crawl-index/
-
+User-agent: *
+Disallow: /staff/
+Disallow: /badgr-server/docs/
```
Comment (ottonomy) [inline apps/mainsite/static/robots.txt]: @jasonKufner I don't think `/badgr-server/docs/` is a path that we use on this application. E.g. https://api.badgr.io/badgr-server/docs/ returns a 404. What is the intent of that change?

Comment (jasonKufner) [reply apps/mainsite/static/robots.txt]: It was listed in the parent ticket(https://jira.concentricsky.com/browse/BP-1246) as a public page:

Public pages badge objects
- Public collections
- Public pathway views (with or without user progress)
- Public (no-auth-required) angular routes
- Forgot password
- Create account
- Sign in
**- badgr-server/docs/***

I can make a pull request to remove it if its not needed

Comment (ottonomy) [reply apps/mainsite/static/robots.txt]: Ah, a space was missing. It meant to say "badgr-server /docs" not "badgr-server/docs". Go ahead and remove from the disallow section, as that is intended to be in the allow section, and access is allowed by default. Only staff is disallow on badgr-server.

Comment (jasonKufner) [reply apps/mainsite/static/robots.txt]: Pushed a commit that removes  badgr-server/docs/

**Pull request merged**